src/styles/utils/_normalize.scss

```
@@ -0,0 +1,9 @@
+
+* {
+  box-sizing: border-box;
+}
+
+.page-content {
+  font-family: Arial, sans-serif;
+  font-size: $font-size;
+}
```

you can put just in html
up to you
```suggestion
html {
  font-family: Arial, sans-serif;
  font-size: $font-size;
}
```